python/lsst/ts/ofc/scripts/generate_alpha_values.py

```diff
@@ -0,0 +1,271 @@
+# This file is part of ts_ofc.
+#
+# Developed for Vera Rubin Observatory.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import pathlib
+from copy import copy
+
+import batoid
+import galsim
+import numpy as np
+import yaml
+from lsst.ts.imsim import OpdMetrology
+from lsst.ts.ofc import OFCData
+from tqdm import tqdm
+
+
+def save_alpha_values(alpha_values: np.ndarray, path: str) -> None:
+    """Save alpha values to a yaml file.
+
+    Parameters
+    ----------
+    alpha_values : np.ndarray
+        Alpha values for the normalized point-source sensitivity.
+    path : str
+        Path to save the alpha values.
+    """
+
+    # Convert numpy array to a list,
+    # as yaml.dump cannot handle numpy arrays directly
+    alpha_values_list = alpha_values.tolist()
+
+    # Complete path to the yaml file
+    yaml_file_path = f"{path}/alpha_values.yaml"
+
+    # Write header and alpha values to the yaml file
+    with open(yaml_file_path, "w") as yaml_file:
+        header = (
+            "---\n"
+            "\n# Default alpha coefficient for the normalized point-source sensitivity\n"
+            "# Normalized point-source sensitivity (PSSN) information\n"
+            "# PSSN ~ 1 - alpha * delta^2\n"
+            "# Eq (7.1) in Normalized Point Source Sensitivity for LSST\n"
+            "# There are 25 terms of alpha value here for z4-z28 to use.\n"
+            "# The procedure to obtain them is detailed in scripts/generate_alpha_values.py\n"
+            "# For more details, please check the page 19-21 in:\n"
+            "# https://docushare.lsst.org/docushare/dsweb/Get/Document-18041/150513.pptx.pdf\n\n"
+        )
+        yaml_file.write(header)
+        yaml.dump(alpha_values_list, yaml_file)
+
+
+def get_alpha_values(
+    config: dict,
+    instrument: str,
+    band: str,
+    znmin: int = 4,
+    znmax: int = 28,
+) -> np.ndarray:
+    """Calculate alpha values for the normalized point-source sensitivity.
+
+    Parameters
+    ----------
+    config : dict
+        Configuration dictionary.
+    instrument : str
+        Instrument name.
+    band : str
+        Filter band.
```

Comment:
Please use the enum instead:
https://github.com/lsst-ts/ts_wep/blob/develop/python/lsst/ts/wep/utils/enumUtils.py#L56-L63

But I saw this:
https://github.com/lsst-ts/ts_ofc/pull/57/files#diff-f30057d41d35ea471822e3136c9a76ddfe777738a514f285607aabc3b1d83b02R243-R247

Therefore, it should be fine.